local-testnet/README.md

```diff
@@ -22,3 +22,12 @@ After a successful run
 * The UI is served at http://localhost:3000/
 * The testnet WS endpoint listens at ws://localhost:9944 and can be used with [Polkadot/Substrate Portal](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9944#/explorer) and [Substrate Contracts UI](https://contracts-ui.substrate.io/?rpc=ws://127.0.0.1:9944).
 
+
+## Clean docker state 
```

Comment:
These are common Docker compose operations. Don't think that they should be included in the README.
But if they are, should be comprehensive and command that you can run "up to start" does not run... and the example should cover the clean up of all the running services, i.e. -f <file1> -f <file2>.
So, for a clean up you can create a simple script like https://github.com/web3labs/epirus-substrate/blob/main/local-testnet/run-all.sh but IMO is not needed.